doc/netplan.md

```diff
@@ -576,6 +576,14 @@ Example:
       enp1s16f1:
         link: enp1
 
+``virtual-functions`` (scalar)
+
+:    (SR-IOV devices only) In certain special cases VFs might need to be
+     configured outside of netplan. For such configurations ``virtual-functions``
+     can be optionally used to set an explicit number of Virtual Functions for
+     the given Physical Function. This should be used for special cases only, as
+     normally netplan allocates as many VFs as there are needed.
```

Comment:
"If unset, the default is to create only as many VFs as are defined in the netplan configuration."